api/hasura/cron/colinksNotificationEmails.ts

```diff
@@ -0,0 +1,125 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { order_by } from '../../../api-lib/gql/__generated__/zeus';
+import { adminClient } from '../../../api-lib/gql/adminClient';
+import { errorResponse } from '../../../api-lib/HttpError';
+import { sendCoLinksNotificationsEmail } from '../../../api-lib/postmark';
+import { verifyHasuraRequestMiddleware } from '../../../api-lib/validate';
+import { isRejected } from '../../../src/common-lib/epochs';
+import { IN_PRODUCTION } from '../../../src/config/env';
+
+export const EMAIL_FOR_REPORTS = 'core@coordinape.com';
+
+async function handler(req: VercelRequest, res: VercelResponse) {
+  if (!IN_PRODUCTION) {
+    return res.status(200).json({ success: true });
+  }
+
+  try {
+    const { profiles } = await adminClient.query(
+      {
+        profiles: [
+          {
+            where: {
+              _and: [
+                { colinks_notification_emails: { _eq: true } },
+                { last_read_notification_id: { _is_null: false } },
+                { links_held: { _gt: 0 } },
+                { links: { _gt: 0 } },
```

Comment:
usually we only check links_held > 0, not total links... prob doo the same here

Reply:
although maybe its better to jsut check links > 0 because the last link cant ever be sold... @CryptoGraffe